example/CMakeLists.txt

```diff
@@ -48,6 +48,11 @@ add_example(glide_computer)
 add_example(hello_units)
 add_example(total_energy)
 
+target_compile_options(glide_computer
+    PRIVATE
+        $<$<CXX_COMPILER_ID:Clang>:-Wno-missing-braces>
+)
+
```

Comment:
Isn't it better to fix the example implementation rather than just disable this warning?

Reply:
I tried adding the suggested braces, and it didn't work (suggested more redundant braces with the same warning). Anyways, this is a bogus warning when the standard permits brace elision. It happened a while back with `std::array`, which I think they manually permitted so as to now warn. It still plagues other types, as you can see in their bugzilla.